cardano-cli/src/Cardano/CLI/EraBased/Query/Run.hs

```diff
@@ -1964,10 +1975,10 @@ writeOutput
   -> b
   -> ExceptT QueryCmdError IO ()
 writeOutput mOutFile content = case mOutFile of
-  Nothing -> liftIO . LBS.putStrLn . encodePretty $ content
+  Nothing -> liftIO . LBS.putStrLn . Aeson.encodePretty $ content
```

Comment:
Why aeson and not `Json.encodeJson`?

Reply:
I was going to do it, but postponing to a future PR to avoid changing too much in the one PR so if something breaks it's easier to pinpoint the problem.